lib/mondo/merchant.rb

```diff
@@ -1,7 +1,7 @@
 module Mondo
   class Merchant < Resource
 
-    attr_accessor :id, :group_id, :logo, :name, :raw_data, :address, :emoji
+    attr_accessor :id, :group_id, :logo, :name, :raw_data, :address, :emoji, :online
```

Comment:
Agreed. Shall sort that now. There are a few other places that we could use it too.